relayer/workers/parachaincommitmentrelayer/parachain-commitment-listener.go

```diff
@@ -14,26 +14,27 @@ import (
 	"golang.org/x/sync/errgroup"
 
 	"github.com/snowfork/polkadot-ethereum/relayer/chain"
+	"github.com/snowfork/polkadot-ethereum/relayer/chain/parachain"
 	chainTypes "github.com/snowfork/polkadot-ethereum/relayer/substrate"
 )
 
-type Listener struct {
-	config   *Config
-	conn     *Connection
+type ParachainCommitmentListener struct {
```

Comment:
Ditto: can we shorten the name of this file and struct to `listener.go` and `Listener` respectively?